src/duct/core/repl.clj

```diff
@@ -17,3 +19,31 @@
   fipp/IOverride
   fipp/IEdn
   (-edn [r] (tagged-literal 'duct/resource (:path r))))
+
+(def ^:private event->file-name
+  (comp #(.getName %)
+        :file))
+
+(def ^:private matches-clojure-file?
+  (partial re-matches #"^[a-zA-Z]{1}[a-zA-Z_]*(.clj|.edn)$"))
+
+(defn- clojure-file?
```

Comment:
Vertical space is important, and we can make this function more space-efficient without sacrificing clarity:

```clojure
(defn- clojure-file? [_ event]
  (-> event event->file-name matches-clojure-file?))
```

In fact, we could actually write it:

```clojure
(defn- clojure-file? [_ {:keys [file]}]
  (re-matches #".(\.clj|\.edn)$" (.getName file)))
```

This gets rid of the previous two functions, and I don't think it's any less clear. Indirection is good for breaking up code, but too much indirection means that the reader needs to jump through more hoops to understand what's going on.